web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx

```diff
@@ -56,27 +56,43 @@ function DefaulterDetailsCard(props: DefaulterDetailsCardProps) {
           </div>
         )}
 
-        <div className="text-success">
+        <div
+          className={church?.fellowshipEquipmentFilledCount === 0 ? '' : 'good'}
+        >
           Fellowship Filled Count: {church?.fellowshipEquipmentFilledCount}
         </div>
         {props.route === 'constituency/fellowship' ? (
-          <div className="text-success">
+          <div
+            className={church.equipmentRecord.pulpits === null ? 'bad' : 'good'}
+          >
             Constituency Filled:{' '}
             {church.equipmentRecord.pulpits === null ? 'No' : 'Yes'}
           </div>
         ) : (
-          <div className="text-success">
+          <div
+            className={
+              church?.constituencyEquipmentFilledCount === 0 ? '' : 'good'
+            }
+          >
             Constituency Filled Count:{' '}
             {church?.constituencyEquipmentFilledCount}
           </div>
         )}
 
-        <div className="text-danger">
+        <div
+          className={
+            church?.fellowshipEquipmentNotFilledCount === 0 ? '' : 'bad'
+          }
+        >
           Fellowship Not Filled Count:{' '}
           {church?.fellowshipEquipmentNotFilledCount}
         </div>
         {props.route === 'constituency/fellowship' ? null : (
-          <div className="text-danger">
+          <div
+            className={
+              church?.constituencyEquipmentNotFilledCount === 0 ? '' : 'bad'
```

Comment:
This being zero should be good rather.

Reply:
tricky one here. we also want that when no forms have been filled, we should see white